src/sentry/features/__init__.py

```diff
@@ -112,6 +112,7 @@
 default_manager.add("organizations:performance-anomaly-detection-ui", OrganizationFeature, True)
 default_manager.add("organizations:performance-chart-interpolation", OrganizationFeature, True)
 default_manager.add("organizations:performance-consecutive-db-issue", OrganizationFeature)
+default_manager.add("organizations:performance-consecutive-http-issue", OrganizationFeature)
```

Comment:
Nit:
```suggestion
default_manager.add("organizations:performance-consecutive-http-detector", OrganizationFeature)
```
would be consistent with more of the detectors, but not a big deal. More importantly, this doesn't match the corresponding `getsentry` PR, because there the flag is called `organization:performance-consecutive-http-issues` with an "s" on the end!

Reply:
Thanks, good call! 👍 
I'll use detector instead of issue(s) instead and also update my other PR accordingly.